sass/apply-a-style-until-a-condition-is-met-with-@while/apply-a-style-until-a-condition-is-met-with-while.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+    <html lang="pt-br">
```

Comment:
the full HTML appears to be indented one level inwards

Reply:
The complete HTML is four levels inward.

With the child elements of the html element which are the head and body with four levels of indentation.

And the child elements of head and body also have four levels of indentation.